packages/statemanager/test/statelessVerkleStateManager.spec.ts

```diff
@@ -43,4 +52,123 @@ describe('StatelessVerkleStateManager: Kaustinen Verkle Block', () => {
       'should have correct codeHash'
     )
   })
+
+  it('put/delete/modify account', async () => {
+    const stateManager = new StatelessVerkleStateManager({ common })
+    stateManager.initVerkleExecutionWitness(block.executionWitness)
+
+    const address = new Address(randomBytes(20))
+
+    try {
+      await stateManager.getAccount(address)
+      assert.fail('should throw on getting account that is not found in witness')
+    } catch (e) {
+      assert.equal(
+        e.message.slice(0, 25),
+        'Missing execution witness',
+        'should throw on getting account that does not exist in cache and witness'
+      )
+    }
+
+    const account = Account.fromAccountData({
+      nonce: BigInt(2),
+    })
+    await stateManager.putAccount(address, account)
+
+    let check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.modifyAccountFields(address, {
+      nonce: BigInt(2),
+    })
+    check = await stateManager.getAccount(address)
+    assert.deepEqual(check, account, 'should return correct account')
+
+    await stateManager.deleteAccount(address)
+
+    check = await stateManager.getAccount(address)
+    assert.isUndefined(check, 'should return undefined for deleted account')
```

Comment:
Minor nit, but instead of re-assigning these to a `check` variable, I would find this more readable if we just inlined the `getAccount` inside of the test comparison for all of these. So something like:

```
 assert.isUndefined(await stateManager.getAccount(address), 'should return undefined for deleted account')
```

Reply:
Fixed.